README.md

```diff
@@ -121,10 +135,10 @@ You can run the greeter from within your desktop session if you add the followin
 
 You have to log out and log back in after adding that line. Then you can run the greeter from command line.
 
-Themes can be opened with a debug console if you set `debug_mode` as `true` inside `/etc/lightdm/web-greeter.yml`. Or, you could run the `nody-greeter` with the parameter `--debug`. I recommend to use the last one, as it is easier and handy.
+Themes can be opened with a debug console if you set `debug_mode` as `true` inside `/etc/lightdm/web-greeter.yml`. Or, you could run the `nody-greeter` with the parameter `--inspect`. I recommend to use the last one, as it is easier and handy.
 
 ```sh
-nody-greeter --debug
+nody-greeter --inspect
```

Comment:
For now, I'd like to keep it as `--debug`. Perhaps in the future I may change it to `--inspect`.

Reply:
I suggested this change because when I run `nody-greeter --debug` I get this:

user@penguin:~$ nody-greeter --debug
electron: [DEP0062]: `node --debug` and `node --debug-brk` are invalid. Please use `node --inspect` and `node --inspect-brk` instead.

when I run nody-greeter --inspect, it works.

Reply:
Yeah, that's an issue with electron that I haven't solved. The recommended command is `nody-greeter -d` instead of `--debug`.

Reply:
Okie, reverted that.